packages/schematics/angular/utility/workspace.ts

```diff
@@ -88,3 +89,28 @@ export async function createDefaultPath(tree: Tree, projectName: string): Promis
 
   return buildDefaultPath(project);
 }
+
+/**
+ * Returns `true` when the workspace root tsconfig is set to use Ivy
+ * as the Angular rendering engine.
+ */
+export function isIvyWorkspace(tree: Tree): boolean {
```

Comment:
I'm not sure this is the correct file for this function since its not related to reading/writing the workspace file.

The function will also not handle extended configurations which would make it problematic to generate an application with ivy enabled in such a scenario.

Reply:
Is it common to have extended root level tsconfig? (Not application specific)?, pretty much in our schematics we only handle `/tsconfig.json` as a base for other tsconfigs.

Re the file, any thoughts were it so go?